mmdet/models/detectors/single_stage.py

```diff
@@ -110,15 +110,21 @@ def simple_test(self, img, img_metas, rescale=False, postprocess=True):
         with no_nncf_trace():
             bbox_results = \
                 self.bbox_head.get_bboxes(*outs, img_metas, self.test_cfg, False)
+            pooled_features = [torch.nn.functional.adaptive_avg_pool2d(feat_map, (1, 1))
+                               for feat_map in x]  # x - backbone+neck output
+            feature_vector = torch.cat(pooled_features, dim=1)
+
+        feature_map = outs[0]  # classification head output
```

Comment:
@goodsong81 I'll also change this part. `outs` is the anchor box prediction not the activation map